src/components/skeletons/ProductCardSkeleton/ProductCardSkeleton.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+import Shimmer from '../Shimmer'
+import ElementSkeleton from '../ElementSkeleton'
+import './product-card-skeleton.scss'
+
+interface Props {
+  bordered?: boolean
+  sectioned?: boolean
```

Comment:
Hm, trying to think a better name for this prop. It means card inside sections like “Just Arrived”, “Deals and Promotions”, right?

maybe `groupedInSection`? 🙈